mplhep/styles_lhcb.py

```diff
@@ -0,0 +1,81 @@
+
+LHCb = {
+    # Plot properties
+    "axes.labelsize": 32,
+    "axes.linewidth": 2,
+    "axes.facecolor": "white",
+    # Custom colors
+    # axes.prop_cycle: cycler('color', ["1f77b4", "ff7f0e", "2ca02c", "d62728", "9467bd", "8c564b", "e377c2", "7f7f7f", "bcbd22", "17becf"]) + cycler('marker', ['o', 's', 'D', '^', 'v', '<', '>', 'P', 'X', '*'])
```

Comment:
why is this uncommented?

Reply:
Because I didn't get around to testing it yet ;)